File: Week 1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // ask user for the height
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    // print
    for (int row = 1; row <= height; row++)
    {
        // print spaces
        for (int col2_space = height - row; col2_space > 0; col2_space--)
        {
            printf(" ");
        }

        // print hashes
        for (int col = 1; col <= 2; col++)
        {
            for (int col2 = 0; col2 != row; col2++)
            {
                printf("#");
            }
            if (col == 1)
            {
                printf("  ");
            }
        }

        // print newline
        printf("\n");
    }
}

File: Week 6/sentimental-mario-more/mario.py
# TODO
from cs50 import get_int

# Prompt user for height
while True:
    height = get_int("Height: ")
    if (height >= 1 and height <= 8):
        break

# Print pyramid
for i in range(height):
    for j in range(2):
        if j == 0:
            if i != height - 1:
                print(" " * (height - i - 1), end="")
            print("#" * (i + 1), end=" " * 2)
        else:
            print("#" * (i + 1), end="")
    print()
